test/studies/hpcc/CommDiags/ra_commDiags.good

```diff
@@ -3,5 +3,5 @@ Bytes per array = 8192
 Total memory required (GB) = 7.62939e-06
 Number of updates = 4096
 
-(get = 0, get_nb = 0, put = 0, put_nb = 0, test_nb = 0, wait_nb = 0, try_nb = 0, execute_on = 212, execute_on_fast = 0, execute_on_nb = 3) (get = 24, get_nb = 0, put = 0, put_nb = 0, test_nb = 0, wait_nb = 0, try_nb = 0, execute_on = 849, execute_on_fast = 1, execute_on_nb = 0) (get = 24, get_nb = 0, put = 0, put_nb = 0, test_nb = 0, wait_nb = 0, try_nb = 0, execute_on = 872, execute_on_fast = 1, execute_on_nb = 0) (get = 24, get_nb = 0, put = 0, put_nb = 0, test_nb = 0, wait_nb = 0, try_nb = 0, execute_on = 837, execute_on_fast = 1, execute_on_nb = 0)
+(get = 0, get_nb = 0, put = 0, put_nb = 0, test_nb = 0, wait_nb = 0, try_nb = 0, execute_on = 212, execute_on_fast = 0, execute_on_nb = 3) (get = 0, get_nb = 0, put = 0, put_nb = 0, test_nb = 0, wait_nb = 0, try_nb = 0, execute_on = 849, execute_on_fast = 1, execute_on_nb = 0) (get = 0, get_nb = 0, put = 0, put_nb = 0, test_nb = 0, wait_nb = 0, try_nb = 0, execute_on = 872, execute_on_fast = 1, execute_on_nb = 0) (get = 0, get_nb = 0, put = 0, put_nb = 0, test_nb = 0, wait_nb = 0, try_nb = 0, execute_on = 837, execute_on_fast = 1, execute_on_nb = 0)
```

Comment:
This and I think a few other .good files you updated also have numa versions. Probably worth checking if we can remove the numa versions now...